### PR TITLE
WIP: Sync latest

### PR DESCRIPTION
**Pull request by awgreene**
This PR was created to test a hunch that #122 is failing due to a bad values.yaml file.

### CODE REVIEW

**Comment by ecordell** [comment]
> error running apply for deployment "openshift-operator-lifecycle-manager/olm-operator" (531 of 710): Deployment.apps "olm-operator" is invalid: spec.template.spec.containers[0].volumeMounts[2].name: Not found: "serving-cert"

File: manifests/0000_50_olm_07-olm-operator.deployment.yaml [hunk 1]
```diff
@@ -23,8 +23,24 @@ spec:
...
         - name: olm-operator
+          volumeMounts:
+            - name: srv-cert
+              mountPath: "/srv-cert"
+              readOnly: true
+            - name: profile-collector-cert
+              mountPath: "/profile-collector-cert"
+              readOnly: true
+            - mountPath: /var/run/secrets/serving-cert
+              name: serving-cert
```
Comment (ecordell) [inline manifests/0000_50_olm_07-olm-operator.deployment.yaml:43]:
```suggestion
            - mountPath: /var/run/secrets/serving-cert
              name: serving-cert
```

Comment (awgreene) [reply manifests/0000_50_olm_07-olm-operator.deployment.yaml:43]: Oh - this should actually be removed from `scripts/olm-deployment.patch.yaml`

**Pull request closed without merging**